### ISSUE TITLE
snactor discover: wrong paths displayed for objects with the same name

### ISSUE THREAD

**Issue by drehak** [oamg/leapp#431]
**Reproducer:**
Create a repository and a tag inside it. Run discover.
```
$ snactor repo new chain
New repository chain has been created in /home/user/chain
$ cd chain/
$ snactor new-tag chain
New tag ChainTag has been created in /home/user/chain/tags/chain.py
$ snactor discover
Repository:
  Name: chain
  Path: /home/user/chain

Actors(0):

Models(0):

Tags(1):
   - ChainTag                            tags/chain.py

Topics(0):

Workflows(0):
```
Create a topic with the same name. Now discover displays a wrong path for our tag.
```
$ snactor new-topic chain
New topic ChainTopic has been created in /home/user/chain/topics/chain.py
$ snactor discover
Repository:
  Name: chain
  Path: /home/user/chain

Actors(0):

Models(0):

Tags(1):
   - ChainTag                            topics/chain.py

Topics(1):
   - ChainTopic                          topics/chain.py

Workflows(0):
```
Similar behavior occurs after creating a model with the very same name.
```
$ snactor new-model --topic ChainTopic chain
New model Chain has been created in /home/user/chain/models/chain.py
$ snactor discover
Repository:
  Name: chain
  Path: /home/user/chain

Actors(0):

Models(1):
   - Chain                               models/chain.py

Tags(1):
   - ChainTag                            models/chain.py

Topics(1):
   - ChainTopic                          models/chain.py

Workflows(0):
```
**Expected output:**
```
$ snactor discover
Repository:
  Name: chain
  Path: /home/user/chain

Actors(0):

Models(1):
   - Chain                               models/chain.py

Tags(1):
   - ChainTag                            tags/chain.py

Topics(1):
   - ChainTopic                          topics/chain.py

Workflows(0):
```
snactor version 0.5.0

**Comment by shaded-enmity** [comment]
@drehak Can you please also check the `--safe` variant of `snactor discover`?

**Comment by drehak** [comment]
Looks alright.
```
$ snactor discover --safe
Repository:
  Name: chain
  Path: /home/user/chain

Actors

Models
  - Chain(Model) ............................................ models/chain.py

Tags
  - ChainTag(Tag) ........................................... tags/chain.py
```

**Comment by vinzenz** [comment]
Fixed in #438
